README.md

```diff
@@ -1 +1,283 @@
-# backend
+
+# hot-coffee-sql
+
+## Learning Objectives
+
+- SQL
+- PostgreSQL
+- CRUD
+- ERD
+
+## Abstract
+
+In this project, you will rewrite the existing [hot-coffee](https://github.com/alem-platform/foundation/tree/main/hot-coffee) project to work with a PostgreSQL database. The main task is to modify the current handlers in hot-coffee so that they execute SQL queries instead of working with JSON.
+
+In addition to refactoring the existing handlers, you will add several new ones related to aggregation and reporting. Throughout this project, you will learn to perform basic SQL operations and explore aggregation tools provided by PostgreSQL.
+
+## Context
+
+Systems become more complex and modernized over time, and the hot-coffee project is no exception. While using a JSON-based database is convenient, it is not scalable and makes maintenance difficult for other developers. Moving to a PostgreSQL database is the better solution.
+
+Fortunately, since the project was initially built using a layered architecture, the transition will be easier. You will need to add a new Data Access Layer (Repositories) and integrate it into the project.
```

Comment:
not add, but **change** Data Access Layer, because they've already implemented it